### PR TITLE
ASCDB

### LINKED ISSUE #1: ASCDB
**Describe the data you'd like**
Upload the ACCDB dataset found [here](https://github.com/peverati/ACCDB).

**Describe ways to obtain the data**
Download data from the ACCDB repository.

**Willing to contribute**
Pier Morgante should be able to help with the ingestion. MolSSI will help with some compute.

**Additional context**

<details><summary>Upload Example</summary>
<p>

```python
import qcportal as ptl
from qcfractal import FractalSnowflake
import pandas as pd

SNOWFLAKE = True

if SNOWFLAKE:
    snowflake = FractalSnowflake()
    client = snowflake.client()
else:
    client = None
print(client)

ds = ptl.collections.ReactionDataset("ASCDB", client=client)

with open("ASCDB.csv", "r") as handle:
    rxns = [x.split(",") for x in handle.read().splitlines()]

gpath = "ASCDB_Geometries"

contrib_name = []
contrib_value = []
for row in rxns[:5]:
    name = row[0]
    rxn = row[1:]

    half = len(rxn) // 2
    molecules = rxn[:half]
    coefs = rxn[half:]

    rxn_data = []
    for mol_name, coef in zip(molecules, coefs):
        mol = ptl.Molecule.from_file(gpath + "/" + mol_name + ".xyz")

        coef = float(coef)
        rxn_data.append((mol, coef))

    rxn = {"default": rxn_data}

    ds.add_rxn(name, rxn)

    contrib_name.append(name)
    contrib_value.append(5)

ds.save()

contrib = {
    "name": "Benchmark",
    "theory_level": "CCSD(T)",
    "values": contrib_value,
    "index": contrib_name,
    "theory_level_details": {"driver": "energy"},
    "units": "hartree",
}
ds.add_contributed_values(contrib)
#ds.save()
```

</p>
</details>

### PR DESCRIPTION
**Pull request by mattwelborn**
## Description
This PR adds ASCDB. Resolves #1.

## Changelog description
Added ASCDB.

## Status
- [x] Ready to go

### CODE REVIEW

**Comment by mattwelborn** [comment]
@dgasmith @PierMorgante Let me know if you see any issues.

File: ASCDB/add_dataset.py [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+else:
+    client = ptl.FractalClient.from_file()
+print(client)
+
+# The new subset you want to add.
+dataset_name = "ASCDB"
+ds = ptl.collections.ReactionDataset(dataset_name, client=client)
+
+# Add the paper
+ds.data.metadata["citations"] = [
```
Comment (dgasmith) [inline ASCDB/add_dataset.py]: Nice thanks for adding.

**Review by dgasmith — COMMENTED** [review]
Ultimately it would be good to have another top level folder that has validation and verification tech, and perhaps another folder with helper CLIs. It would be good to put changes in a folder called `modifications`, `commits`, or `revisions`? Something of this sort.

All modifications would follow the form of YYYY-MM-DD-ASCDB which contain change scripts.

The script looks good!

FYI @loriab

**Comment by PierMorgante** [comment]
> @dgasmith @PierMorgante Let me know if you see any issues.

Let me know if I can be of any help.

**Review by dgasmith — APPROVED** [review]
Looks good, lets try this as a first round modification.

**Comment by mattwelborn** [comment]
Who executes the script? The PR author or the merger?

**Comment by dgasmith** [comment]
The author doesn't always work since it could be a 3rd party. We might want to have a "staged" folder in the future which would contain the unexecuted scripts which could be merged at a later date? We could also look into automated infrastructure to run on merge, but that seems a bit dangerous.

Would you mind running this one?

**Comment by mattwelborn** [comment]
Done.

**Pull request merged**